docs/rfcs/XeGPU.md

```diff
@@ -12,6 +14,9 @@ XeGPU operations are introduced when there is a special Xe instruction not model
 load and store. In some cases, one XeGPU op may lower to a sequence of instructions for a dedicated and performance-critical function.
 For example, create_tdesc is mapped to a fixed sequence of instructions to create an address description.
 
+The operation definition is general and works for workgroup, subgroup, or work item level. When working at workgroup level, the operation must 
+attach `wg_map` attribute, and work item level operation must attach `sg_map` attribute. 
+
 Below is a summary.
 
```

Comment:
nit: in the following, `XeGPU op is carried out by all the work items within a subgroup` needs to be updated too.